app/views/components/_chart.html.erb

```diff
@@ -21,13 +21,14 @@
         1 => {lineWidth: 2, lineDashStyle: [10, 2], color: '#454a4c'}
     }
   }
-  chart_format_data = rows.each_with_object([]) do |row, acc|
-    acc << {
-    name: row[:label],
-    linewidth: 10,
-    data: keys.zip(row[:values])
-    }
-  end
+  rows.length > 0 && keys.length > 0
```

Comment:
I think we might be missing an `if` here?

Reply:
yep, will fix